submissions/nadsatt/friends-app/modules/search-filter-items.js

```diff
@@ -0,0 +1,32 @@
+import { SearchFilter, ToggleFilter } from './filter-items.js';
+
+class UserNameSearchFilter extends SearchFilter {
+    constructor(userService){
+        super('name', userService);
+    }
+}
+
+class UserEmailSearchFilter extends SearchFilter {
+    constructor(userService){
+        super('email', userService);
+    }
+}
+
+class UserLocationSearchFilter extends SearchFilter {
+    constructor(userService){
+        super('location', userService);
+    }
+}
+
+class GenderToggleFilter extends ToggleFilter {
+    constructor(userService){
+        super('female', 'male', 'gender', userService);
+    }
+}
+
+export default [
```

Comment:
And why this an array? It is very inconvenient way to export :) Just export as named entities and create array where you need it.